core/runtime.rs

```diff
@@ -2056,20 +2082,16 @@ impl JsRuntime {
   pub async fn load_main_module(
     &mut self,
     specifier: &ModuleSpecifier,
-    code: Option<String>,
+    code: Option<ModuleCode>,
   ) -> Result<ModuleId, Error> {
     let module_map_rc = Self::module_map(self.v8_isolate());
     if let Some(code) = code {
       let scope = &mut self.handle_scope();
       module_map_rc
         .borrow_mut()
         .new_es_module(
-          scope,
-          // main module
-          true,
-          specifier.as_str(),
-          code.as_bytes(),
-          false,
+          scope, // main module
+          true, specifier, &code, false,
```

Comment:
Could you bring back the old formatting?

Reply:
Let me take a look - I suspect that `rustfmt` will give me a bit of trouble here. I can at least move the comment to the appropriate parameter.